src/extension.ts

```diff
@@ -71,6 +72,14 @@ export async function activate(context: vscode.ExtensionContext) {
 	);
 	const userService = new UserService(globalStateTokenStorage);
 
+	const accessToken = userService.getLinkedToken();
+
+	vscode.commands.executeCommand(
+		'setContext',
```

Comment:
can you explain this string literal here?

Reply:
https://code.visualstudio.com/api/references/when-clause-contexts#add-a-custom-when-clause-context

Reply:
Nice, actually I wasn't aware of this feature.